### DISCUSSION TITLE
Apps always show as Spotify in menu

### DISCUSSION THREAD

**Discussion by flex1981** [BimmerGestalt/AAIdrive#371] [Q&A]
Hi,

Amazing app btw ! Only discovered it a few days ago but it works really well, I just wanted to feedback that it the only issue I have noticed is my apps always appear as Spotify in the media list, once in that app I can select the different ones from there. 

Is this by design? I noticed there was an option to always show as Spotify but this is not enabled.

**Comment by hufman** [comment]
Thank you for the kind words!
Indeed, that is the expected behavior: Spotify is the only real app in the list, and so is the only entry that can show the Now Playing checkmark. The other apps are technically placeholder icons, for the car to trigger a full app to be launched when they are selected, but there aren't any official BMW app resources for those apps and so AAI can only change the music playback and then shows the Spotify interface.

The `Always Show As Spotify` option is for users who don't have Spotify installed but would still prefer the Spotify interface which enables global coverart. It would then confusingly show Spotify in the music list, even though Spotify is not installed, so it's not enabled by default.
